### ISSUE TITLE
Error this.rules.map is not a function

### ISSUE THREAD

**Issue by rendomnet** [DrKain/tidy-url#8]
I'm using it in Chrome extension.

this command gives error.
let tidy = TidyURL.clean('https://google.com');

```
TypeError: this.rules.map is not a function
    at TidyCleaner.get (popup.js:156049:31)
    at TidyCleaner.__webpack_modules__.../node_modules/tidy-url/lib/index.js.TidyCleaner.clean
```

```
Object.defineProperty(TidyCleaner.prototype, "expandedRules", {
        get: function () {
            return this.rules.map(function (rule) {
                return Object.assign({ rules: [], replace: [], redirect: '' }, rule);
            });
        },
        enumerable: false,
        configurable: true
    });
```

**Comment by rendomnet** [comment]
@DrKain what can cause this? Every other module working without problem.
"webpack": "^5.68.0"
Maybe I should message webpack devs?

**Comment by DrKain** [comment]
I'm honestly not sure. I don't use webpack and I simply don't have the time to look into this issue right now, this tool was also not designed to be used in the way you're trying. I'll take another look at this issue if I find the time another day, but currently it's low priority. You're more than welcome to contact the devs and ask what the issue is.

**Comment by rendomnet** [comment]
@DrKain I have located the source of issue.

this line in rules js 
`var module = module ?? { exports: null };`
When this line is removed then everything is working.

**Comment by DrKain** [comment]
Resolved in #9
